spec/routes_spec.rb

```diff
@@ -2,14 +2,14 @@
 
 describe "routes", type: :request do
   it "redirects to s3" do
-    get "/translations/files/test.zip"
-    expect(response).to redirect_to("https://mybucket.s3.us-east-1.amazonaws.com/translations/files/test.zip")
+    get "/#{Package::TRANSLATION_FILES_PATH}test.zip"
```

Comment:
this one would also be distinct.

There are 2 distinct paths that happen to be the same, but don't necessarily need to be.
1. The path used for the mobile-content-api route
2. The path in S3 that you store the files in